packages/e2e/cypress/e2e/common/about.cy.js

```diff
@@ -14,7 +14,7 @@ limitations under the License.
 describe('About page', () => {
   before(() => {
     cy.request('/v1/properties')
-      .then(({ body }) => body)
+      .its('body')
```

Comment:
Seems to be a minor lint issue that the tests are complaining about

Should use:
```
      cy.request('/v1/properties').its('body').as('installProperties');
```

Reply:
Thanks, pushed an update